src/reaper_osara.cpp

```diff
@@ -1261,6 +1261,11 @@ void postCursorMovementScrub(int command) {
 		fakeFocus = FOCUS_RULER; // Set this even if we aren't reporting.
 }
 
+void postInvertSegmentScrubRange(int command) {
+	// Translators: Reported when users invert the looped or one shot segment offsets.
+	outputMessage(translate("inverted segment offsets"));
```

Comment:
Is there a reason this isn't defined in `POST_COMMAND_MESSAGES` instead?

Reply:
> Is there a reason this isn't defined in POST_COMMAND_MESSAGES instead?
> 
Nope, I'm just stupidier at that time of night/morning. Moving that and merging now.